generators/client/templates/vue/src/test/javascript/e2e/modules/account/account.spec.ts.ejs

```diff
@@ -393,11 +393,13 @@ describe('Account', () => {
         userManagementPage = await navBarPage.getUserManagementPage();
 
         const deletePage: UserManagementDeletePage = await userManagementPage.getDeletePage('user_test');
+        await waitUntilDisplayed(deletePage.modal);
```

Comment:
I think this is not required as `getDeletePage()` call waits for an element `okButton` to be clickable

Reply:
This is now the same as the entity spec. Without that call the spec failed with the ok button not clickable, although the fake user was deleted, but can try without that again locally.